Phases/Assessments/Phase_1.md

```diff
@@ -0,0 +1,37 @@
+# Phase 1 Assessment
+
+The phase 1 assessment will test your ability to write basic JavaScript programs for the Node.js platform as well as your fundamental SQL knowledge.
+
+You will _not_ be assessed on your HTML, CSS, or other web-specific skills.
+
+## Skills Assessed
+
+- Programming (Beginner Level)
+- JavaScript (Beginner Level)
+- Node.js (Beginner Level)
+- Testing (Beginner Level)
+- SQL (Beginner Level)
+
+Some examples of specific things you _may_ be required to do as part of the assessment (this is **not** an exhaustive list):
+
+- Manipulate Arrays, Strings, and Objects in JavaScript
+- Use conditionals and iteration to solve a problem
+- Pull values out of a JSON object
+- Write tests for your code with Mocha and Chai
+- Use synchronous functions from the Node.js standard library
+- Read and write to files from a Node.js program
+- Design a relational database schema
+- Write SQL to query a small PostgreSQL database (1-4 tables)
```

Comment:
I think it's awesome that were listing what is and isnt going to be on the assessment. There isn't an infinite amount of things phase 1 is intending to teach. There is an explicit, objective and finite list [here](https://github.com/LearnersGuild/v1.5-curriculum/blob/master/phases/1/skills.md)